Rakefile

```diff
@@ -7,3 +7,14 @@ RSpec::Core::RakeTask.new(:spec)
 
 task build: ["rubocop:auto_correct", :spec]
 task default: :build
+
+desc "Run acceptance tests with vagrant-spec"
```

Comment:
Mind adding a comment that/why you're bringing up fake "remote" VMs in system-vagrant then testing against them in bundle-vagrant?